oct_sim_analysis/modplot.py

```diff
@@ -0,0 +1,208 @@
+         xlabel='', ylabel=r'Probability Density', ylabel_ratio='Ratio to\nTruth', units='', 
+         xlim=(0,1), ylim=(0,1), ylim_ratio=(0.7,1.3),
+         xticks=None, yticks=None, xtick_step=None, ytick_step=None, ytick_ratio_step=0.15,
+         **kwargs):
+    
+    # gridspec options
+    gridspec_kw = {'height_ratios': (3.5, 1) if ratio_plot else (1,), 'hspace': 0.0}
+    if isinstance(gridspec_update, dict):
+        gridspec_kw.update(gridspec_update)
+
+    # get subplots
+    nsubplots = 2 if ratio_plot else 1
+    fig, axes = plt.subplots(nsubplots,  gridspec_kw=gridspec_kw, figsize=figsize)
+    if nsubplots == 1:
+        axes = [axes]
+        
+    # axes limits
+    for ax in axes:
+        ax.set_xlim(*xlim)
+    axes[0].set_ylim(*ylim)
+    if ratio_plot:
+        axes[1].set_ylim(*ylim_ratio)
+        
+    # axes labels
+    if units:
+        xlabel = r'{} [{}]'.format(xlabel, units)
+        ylabel = r'{} [{}{}]'.format(ylabel, units, r'$^{-1}$')
+    axes[-1].set_xlabel(xlabel)
+    axes[0].set_ylabel(ylabel)
+    if ratio_plot:
+        axes[1].set_ylabel(ylabel_ratio, fontsize=8)
+        
+    # tick settings
+    for ax in axes:
+        ax.minorticks_on()
+        ax.tick_params(top=True, right=True, bottom=True, left=True, direction='in', which='both')
+    if ratio_plot:
+        axes[0].tick_params(labelbottom=False)
+        axes[1].tick_params(axis='y', labelsize=8)
+    
+    # tick locations and labels
+    if xtick_step is not None:
+        xticks_locs = [round(xlim[0] + i*xtick_step, 4) for i in range(1+math.floor((xlim[1]-xlim[0])/xtick_step))]
+        axes[-1].set_xticks(xticks_locs)
+        if xticks is None:
+            axes[-1].set_xticklabels(list(map(str, xticks_locs)))
+        else:
+            axes[-1].set_xticklabels(xticks)
+    if ytick_step is not None:
+        yticks_locs = [round(ylim[0] + i*ytick_step, 4) for i in range(1+math.floor((ylim[1]-ylim[0])/ytick_step))]
+        axes[0].set_yticks(yticks_locs)
+        if yticks is None:
+            axes[0].set_yticklabels(list(map(str, yticks_locs)))
+        else:
+            axes[0].set_yticklabels(yticks)
+    if ytick_ratio_step is not None and ratio_plot:
+        rangeVal = int(round((ylim_ratio[1]-ylim_ratio[0])/ytick_ratio_step))
+        yticks = [round(ylim_ratio[0] + i*ytick_ratio_step, 4)
+                  for i in range(1+rangeVal)][1:-1]
+        axes[1].set_yticks(yticks)
+        axes[1].set_yticklabels(list(map(str, yticks)))
+    
+    return fig, axes
+
+# general function to handle the style
+def style(func, **kwargs):
+    if func == 'errorbar':
+        defaults = {'fmt': 'o', 'lw': 1.5, 'capsize': 1.5, 'capthick': 1, 'markersize': 1.5}
+    elif func == 'plot':
+        defaults = {'lw': 1.5, 'ls': '-'}
+    else:
+        raise ValueError('unrecognized plot function {}'.format(func))
+    defaults.update(kwargs)
+    return defaults
+
+# default style for cms data
+def cms_style(*args, **kwargs):
```

Comment:
Same comment for this - seems like extraneous utilities

Reply:
You're right so I removed this entire folder. This is personal analysis stuff that I'll keep on my fork in a dev branch later.